integr8ly/integreatly-operator
# replaced parameters to quicken scrape duration

[CSSRE-830](https://issues.redhat.com/browse/CSSRE-830.) for the observed issue in RHMI 1.x which caused intermittent scrape timeouts and hence intermittent alerts.The following fixes this in RHMI2.x https://issues.redhat.com/browse/INTLY-6406

Before: (high 3 -4)
![no](https://user-images.githubusercontent.com/53817495/77163960-24573880-6aa7-11ea-829b-40b82617cd21.png)
After: (low 1-2)
![Screenshot from 2020-03-23 11-33-36](https://user-images.githubusercontent.com/53817495/77312587-2e24aa00-6cfa-11ea-8fd6-9800f8a6b8c8.png)
### Prerequisite:
OSD cluster
### Verification steps:
1. Install integreatly using this branch
2. when everythngs up run the following expression in Prometheus : `scrape_duration_seconds{job="openshift-monitoring-federation"}`
3. The graph should look like the after graph above.
4. Check that the results of the following expressions are using the rhmi namespace:
`kubelet_volume_stats_used_bytes`
`kubelet_volume_stats_available_bytes`
